kolibri/plugins/registry.py

```diff
@@ -48,14 +67,15 @@ def initialize():
             try:
                 plugin_module = importlib.import_module(app + ".kolibri_plugin")
                 logger.debug("Loaded kolibri plugin: {}".format(app))
+                all_classes = dict([(name, cls) for name, cls in plugin_module.__dict__.items() if isinstance(cls, type)])
                 plugin_classes = []
-                for obj in plugin_module.PLUGINS:
+                for obj in all_classes.values():
                     if type(obj) == type and issubclass(obj, KolibriPluginBase):
                         plugin_classes.append(obj)
                 for plugin_klass in plugin_classes:
-                    plugin_obj = plugin_klass()
-                    for hook, callback in plugin_obj.hooks().items():
-                        hooks.register_hook(hook, callback)
+                    # Initialize the class, nothing more happens for now.
+                    logger.debug("Initializing plugin: {}".format(plugin_klass.__name__))
+                    plugin_klass()
```

Comment:
I think that in our examples of hooks and plugins, we won't show plugin classes or hooks with custom `__init__` constructors. But for flexibility, as I said about the hooks, I think it's great that they actually do get instantiated because it gives an additional power. But luckily it's something that we can leave out for most parts.

Instantiating an object that doesn't perform anything in its constructor and only holds a few static references in its attributes should be quite easy resource-wise.